modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators.Sample/MustBeVariantSamples.cs

```diff
@@ -136,6 +136,256 @@ public void MethodWithAttributeOk()
     {
     }
 
+    [GenericTypeAttribute<bool>()]
+    public void MethodWithAttributebool()
```

Comment:
I find these names hard to read, could you use the name of the type instead of the keyword? For example:

```
bool -> Bool
int -> Int32
long -> Int64
float -> Single
double -> Double
```